src/main/kotlin/registration/RegistrationServiceEndpoint.kt

```diff
@@ -45,24 +45,24 @@ class RegistrationServiceEndpoint(
         server.start(wait = false)
     }
 
-    private fun responseError(code: HttpStatusCode, reason: String): registration.Response {
+    private fun responseError(code: HttpStatusCode, reason: String): Response {
         logger.warn { "Response has been failed. $reason" }
-        return registration.Response(code, "Response has been failed. $reason")
+        return Response(code, "Response has been failed. $reason")
     }
 
-    private fun onPostRegistration(name: String?, pubkey: String?): registration.Response {
+    private fun onPostRegistration(name: String?, pubkey: String?): Response {
         if (name == null)
             return responseError(HttpStatusCode.BadRequest, "Parameter \"name\" is not specified.")
         if (pubkey == null)
```

Comment:
You don't check the case then they are both null. 
It is not a part of your PR, but I think we could leave a TODO here

Reply:
why do we need to check it?

Reply:
For example if I submit a form, I would like to know all errors at once, not fixing one and then getting the information that something else is also wrong